src/util/tapecloud_utils.js

```diff
@@ -0,0 +1,86 @@
+/* Copyright (C) 2023 NooBaa */
+
+'use strict';
+
+const { exec } = require("child_process");
+const util = require('util');
+const config = require('../../config');
+
+const asyncexec = util.promisify(exec);
+
+const ERROR_DUPLICATE_TASK = "GLESM431E";
+
+async function eeadm_failure_handler(error) {
+    const { stdout } = error;
+
+    // Find the line in the stdout which has the line 'task ID is, <id>' and extract id
+    const match = stdout.match(/task ID is (\d+)/);
+    if (match.length !== 2) {
+        throw error;
+    }
+
+    const task_id = match[1];
+
+    // Fetch task status and see what failed
+    const { stdout: taskshowstdout } = await asyncexec(`eeadm task show -r ${task_id}`);
```

Comment:
this command (eeadm task show) should also be in its own wrapped script so we don't have to hard coded it here.

Reply:
When you say script, do you mean another JS file dedicated to this or a string which simply stores this command?